torch/_torch_docs.py

```diff
@@ -2971,6 +2971,72 @@ def parse_kwargs(desc):
             [ 2.]])
 """)
 
+add_docstr(torch.frobenius_norm,
+           r"""
+.. function:: frobenius_norm(input) -> Tensor
+
+Returns the frobenius norm of the :attr:`input` tensor.
+
+.. math::
+    ||x||_{F} = \sqrt{tr(xx^{H})}
+
+Args:
+    input (Tensor): the input tensor, should be a matrix (2-D tensor)
+Example::
+
+    >>> a = torch.randn(2, 3)
+    >>> a
+    tensor([[ 1.0043,  0.5987, -1.2026],
+            [ 1.7241, -2.6561,  0.8088]])
+    >>> a.frobenius_norm()
+    tensor(3.6735)
+
+.. function:: frobenius_norm(input, dim, keepdim=False) -> Tensor
```

Comment:
I'd rather dispatch to theses functions in `norm` (i.e., what matlab and numpy do) rather than document them separately.